sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceivedMessage.java

```diff
@@ -84,6 +86,15 @@ public BinaryData getBody() {
         }
     }
 
+    /**
+     * Gets the actual payload/data wrapped by the {@link ServiceBusReceivedMessage}.
+     *
+     * @return A byte array representing the data.
+     */
+    public byte[] getBodyAsBytes() {
+        return getBody().toBytes();
+    }
```

Comment:
What happens if the AmqpBodyType is SEQUENCE or VALUE?

Reply:
We will design with other languages,
But I see we can have something like  
public AmqpSequenceBody getBodyAsSequence();
and in getBodyAsBytes we continue to throw for any type other than Types=DATA  which is bytes in AMQP.  t